### PR TITLE
OBSDOCS-1097: Add more two callouts in the File Storage Extension section

### PR DESCRIPTION
**Pull request by max-cx**
Version(s):

4.12, 4.13, 4.14, 4.15, 4.16, 4.17

Issue:

https://issues.redhat.com/browse/OBSDOCS-1097

Link to docs preview:

https://79739--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/otel/otel-collector/otel-collector-extensions.html#filestorage-extension_otel-collector-extensions

QE review:
- N/A (the YAML block was previously reviewed and approved by QE)

Additional information:

### CODE REVIEW

**Review by frzifus — APPROVED** [review]

File: observability/otel/otel-collector/otel-collector-extensions.adoc [hunk 1]
```diff
@@ -142,10 +142,10 @@ include::snippets/technology-preview.adoc[]
     exporters:
       otlp:
         sending_queue:
-          storage: file_storage/all_settings
+          storage: file_storage/all_settings # <7>
 
     service:
-      extensions: [file_storage/all_settings]
+      extensions: [file_storage/all_settings] # <8>
```
Comment (dfitzmau) [inline observability/otel/otel-collector/otel-collector-extensions.adoc:148]: The brackets are intended in this entry and not the other entry?

File: observability/otel/otel-collector/otel-collector-extensions.adoc [hunk 2]
```diff
@@ -158,6 +158,8 @@ include::snippets/technology-preview.adoc[]
 <4> Specifies the directory in which the compactor stores the telemetry data.
 <5> Defines the maximum size of the compaction transaction. To ignore the transaction size, set to zero. If omitted, the default is `+65536+` bytes.
 <6> When set, forces the database to perform an `fsync` call after each write operation. This helps to ensure database integrity if there is an interruption to the database process, but at the cost of performance.
+<7> Buffers the OTLP Exporter data on the local file system.
```
Comment (dfitzmau) [inline observability/otel/otel-collector/otel-collector-extensions.adoc:161]: This suggestion is outside the primary changes, but consider defining what OTLP equals to the document: OpenTelemetry Protoco

**Review by dfitzmau — APPROVED** [review]
/remove-label peer-review-in-progress

/label peer-review-done

File: observability/otel/otel-collector/otel-collector-extensions.adoc (hunk 1 above)
Comment (max-cx) [reply observability/otel/otel-collector/otel-collector-extensions.adoc:148]: @frzifus, ^ this is a question for you.

File: observability/otel/otel-collector/otel-collector-extensions.adoc (hunk 2 above)
Comment (max-cx) [reply observability/otel/otel-collector/otel-collector-extensions.adoc:161]: Thank you, I've deabbreviated OTLP in the section's text.

File: observability/otel/otel-collector/otel-collector-extensions.adoc (hunk 1 above)
Comment (frzifus) [reply observability/otel/otel-collector/otel-collector-extensions.adoc:148]: Yes, that entry is a list where multiple extensions can be registered.

**Pull request merged**